app/src/main/java/org/mozilla/focus/fragment/FirstrunFragment.java

```diff
@@ -23,7 +24,13 @@
     public static final String FIRSTRUN_PREF = "firstrun_shown";
 
     public static FirstrunFragment create() {
-        return new FirstrunFragment();
+        final Fade exitFade = new Fade();
+        exitFade.setDuration(200);
```

Comment:
Why not an XML animation? It's often easier to tweak and add additional effects. :)